### PR TITLE
8.0 - Porting 'base_contact' module

### PR DESCRIPTION
**Pull request by sebalix**
- Cherry picked commit 0f05753fad2c351cee80747054a09cf0b8cad190 from the 7.0 branch,
- Cherry picked translations (DE+FR) from the 7.0 branch (commits 9862d7bdfc3fda08d0ad061c3c3cf0cc34df98ec , 312ac916e622c19f26bae56793ae33882203a3ff and 0c346e4f15905ff020f5e083d2119ed60d0ef5bb)
- Removed `process` and `contacts` modules (`process` does not exist anymore on 8.0, and I didn't find any requirement for `contacts`),
- Module description in a new README.rst file
- Fix unit tests: name of the company renamed from 'Your Company' to 'YourCompany'

### CODE REVIEW

**Comment by sebalix** [comment]
Build failed on the `partner_firstname` module, it could be related to `base_contact`, I will attempt to reproduce this locally.

**Comment by sebalix** [comment]
Tests fail locally on a database with only `partner_firstname` installed, no link with `base_contact`.

File: base_contact/README.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This module allows you to manage your contacts
+==============================================
+
+Features
+--------
+    - a contact can have several job positions in different companies,
+    - birthdate and nationality can be defined.
```
Comment (yvaucher) [inline base_contact/README.rst]: This porting would be an opportunity to use OCA readme template:
https://github.com/OCA/maintainers-tools/blob/master/template/module/README.rst

File: base_contact/__openerp__.py [hunk 2]
```diff
@@ -26,19 +26,8 @@
     'website': 'http://www.openerp.com',
     'category': 'Customer Relationship Management',
     'complexity': "expert",
-    'description': """
-This module allows you to manage your contacts
```
Comment (yvaucher) [inline base_contact/__openerp__.py:30]: You might want to put this phrase in `'summary'` of in `__openerp__.py` just bellow `'name'` key

**Comment by yvaucher** [comment]
Just 2 remarks to have this module better displayed in module list and on apps.odoo.com

File: base_contact/__openerp__.py (hunk 2 above)
Comment (pedrobaeza) [inline base_contact/__openerp__.py:30]: I would put in the summary the sentence "Manage your contacts separately".

File: base_contact/README.rst (hunk 1 above)
Comment (sebalix) [inline base_contact/README.rst]: I was not sure about the content of 'Installation' and 'Configuration' sections.
Also, there are others contributors to add, in Git I have found TheCloneMaster, Sandy Carter, EL HADJI DEM and Xavier ALT from OpenERP. Should I add all of them?

File: base_contact/__openerp__.py (hunk 2 above)
Comment (sebalix) [inline base_contact/__openerp__.py:30]: Done.

**Comment by yvaucher** [comment]
Error is still the error of partner_firstname module unrelated to this PR

Thanks for the changes.

:+1:

File: base_contact/README.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ * Migrate to the new API
+
+Credits
+=======
+
+Contributors
+------------
+
+* OpenERP SA
+* Sebastien Alix <sebastien.alix@osiell.com>
```
Comment (pedrobaeza) [inline base_contact/README.rst:36]: Yes, please, add the other contributors as this is the first time they are summarized. Open source will reward you :wink:

Comment (sebalix) [inline base_contact/README.rst:36]: Updated

**Comment by sebalix** [comment]
I just cherry picked translations (DE+FR) from the 7.0 branch (commits 9862d7bdfc3fda08d0ad061c3c3cf0cc34df98ec , 312ac916e622c19f26bae56793ae33882203a3ff and 0c346e4f15905ff020f5e083d2119ed60d0ef5bb)

File: base_contact/i18n/de.po [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+msgstr "Mobil:"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "All partner positions"
+msgstr "Alle Positionen des Partners"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "{'invisible': [('contact_type','=','attached')]}"
```
Comment (pedrobaeza) [inline base_contact/i18n/de.po:154]: Better remove this line that can be conflictive (it's a bug in the translation export)

File: base_contact/i18n/base_contact.pot [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "All partner positions"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "{'invisible': [('contact_type','=','attached')]}"
```
Comment (pedrobaeza) [inline base_contact/i18n/base_contact.pot:150]: Better remove this line that can be conflictive (it's a bug in the translation export)

File: base_contact/i18n/fr.po [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+msgstr "Portable :"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "All partner positions"
+msgstr "Tous les contacts"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "{'invisible': [('contact_type','=','attached')]}"
```
Comment (pedrobaeza) [inline base_contact/i18n/fr.po:152]: Better remove this line that can be conflictive (it's a bug in the translation export)

**Comment by pedrobaeza** [comment]
Some minor issues. The rest LGTM.

**Comment by guewen** [comment]
:+1: 
I removed the erroneous po lines before the merge.

**Comment by sebalix** [comment]
Thank you!

**Pull request merged**